### PR TITLE
Configure Emails in Admin Backoffice

### PR DESCRIPTION
**Pull request by RicardoVaz1**
Why:
* https://3.basecamp.com/4319812/buckets/26581088/todos/5227038325

How:
* Creating a Email controller in Active Admin

### CODE REVIEW

File: app/controllers/emails_controller.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+# Controller that manages emails
+class EmailsController < ApiController
+  def index
+    render json: Email.all
+  end
+
+  def create
+    email = Email.create(email_params)
+    render json: email
+  end
```
Comment (zediogoviana) [inline app/controllers/emails_controller.rb:12]: If you are moving this to be configurable in the backend, why doesn't it also exist an update? I assume that there is a good reason for this to be configurable this way, and not just have them in the code...

File: app/admin/emails.rb [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+  #
+  # or
+  #
+  # permit_params do
+  #   permitted = [:email_type, :subject, :body]
+  #   permitted << :other if params[:action] == 'create' && current_user.admin?
+  #   permitted
+  # end
+end
```
Comment (zediogoviana) [inline app/admin/emails.rb:20]: Are all of these comments supposed to be here? The same thing for the other files that have comments. I don't think this adds any valuable information.

File: app/controllers/emails_controller.rb (hunk 1 above)
Comment (RicardoVaz1) [reply app/controllers/emails_controller.rb:12]: The Active Admin has his own controller that have already an update function. I'm just using that create function to create the default templates in the seeds file

EmailsAdminBackffice

Comment (zediogoviana) [reply app/controllers/emails_controller.rb:12]: Oh, right! I remember now 😅

File: app/admin/emails.rb (hunk 2 above)
Comment (RicardoVaz1) [reply app/admin/emails.rb:20]: No, I'll delete them

**Review by zediogoviana — APPROVED** [review]
👌

**Pull request merged**